core/src/com/projectkorra/projectkorra/BendingPlayer.java

```diff
@@ -354,36 +354,23 @@ public static OfflineBendingPlayer getOrLoadOffline(@NotNull final OfflinePlayer
 		}
 	}
 
-	/**
-	 * Gets a BendingPlayer instance for the provided player
-	 * @param player The player
-	 * @return The BendingPlayer instance
-	 */
-	public static BendingPlayer getBendingPlayer(@NotNull final Player player) {
-		return getBendingPlayer((OfflinePlayer)player);
-	}
-
-
```

Comment:
This constructor is deleted because Player extends OfflinePlayer so the constructor that this references will accept the Player instance anyways, making this redundant